uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences-my-leave-calendar.js

```diff
@@ -1,9 +1,91 @@
+/* globals jQuery */
+
 var page = require('./page');
 
 module.exports = (function () {
   return page.extend({
     /**
-     * Wait for the page to be ready by looking at 
+     * Clears all selected months from the calendar "Selected Months" field.
+     *
+     * @returns {Object} - returns a reference to itself.
+     */
+    clearAllSelectedMonths: function () {
+      this.casper.evaluate(function () {
+        jQuery('.chr_leave-calendar__day-selector .close.ui-select-match-close')
+          .each(function () {
+            jQuery(this).click();
+          });
+      });
+
+      return this;
+    },
+
+    /**
+     * Display all months for the leave calendar by clearing the
+     * "Selected Months" field.
+     *
+     * @returns {Object} - returns a reference to itself.
+     */
+    showAllMonths: function () {
```

Comment:
Are we using this method?

Reply:
I didn't end up using it, but it could be useful for other tests. Should I remove it?

Reply:
Yes please remove it. No need to keep unused code.